### PR TITLE
Mail fix

### PR DESCRIPTION
**Pull request by palmieric**
This PR update the mail tests to use ad-hoc application and service

### CODE REVIEW

**Comment by mganisin** [comment]
Commits need refactor (likely squash)

**Review by mganisin — APPROVED** [review]

File: testsuite/tests/system/messages/mail_templates.yml [hunk 1]
```diff
@@ -20,20 +20,20 @@ subject_templates:
...
+      <application>  Description: <app_description>To view application, follow this link: https://3scale-admin.<threescale_superdomain>/p/admin/applications/[0-9]+You
       are receiving this because you subscribed to this notification.View notification
       preferences on https://3scale-admin.<threescale_superdomain>/p/admin/user/notification_preferences----==_mimepart_.+Content-Type:
       text/html; charset=UTF-8Content-Transfer-Encoding: quoted-printable<!DOCTYPE
       html><html lang=3D"en"><head><meta charset=3D"utf-8" /></head><body style=3D"font-family:
       &#39;Lucida Grande&#39;, verdana, sans-serif;"><p>Dear <username>,</p><p>A new
       application subscribed to the <aplan> plan on the <service> service
-      of the <test_group> account.</p><p><b>Application details:</b></p><ul><li><b>Name:</b>
-      API signup</li><li><b>Description:</b> API signup</li></ul><p><a href=3D"https://3scale-admin.<threescale_superdomain>/p/admin/applications/[0-9]+">View
+      of the <test_group> account.</p><p><b>Application details:</b></p><ul><li><b>Name:</b>
```
Comment (mdujava) [inline testsuite/tests/system/messages/mail_templates.yml:35]: introducing trailing space

Comment (palmieric) [reply testsuite/tests/system/messages/mail_templates.yml]: removed

**Review by mdujava — APPROVED** [review]

**Comment by mganisin** [comment]
Multiple commits, likely squash needed

**Review by mganisin — CHANGES_REQUESTED** [review]
squash commits please

**Review by mganisin — APPROVED** [review]

**Pull request merged**